code/game/machinery/OpTable.dm

```diff
@@ -86,7 +86,7 @@
 	if(!ishigherbeing(user) && !isrobot(user)) //No ghosts or mice putting people into the sleeper
 		return
 	var/mob/living/L = O
```

Comment:
Please change this this to `/mob/living/carbon/human/H` : right now this lets silicons but themselves on top of tables.

Reply:
good now?